mineral-ui/mineral-ui
# [Website] Increase maxHeight of codeblocks

### Description

Make sure the codeblock maxHeight fits the content of the Usage source in Getting Started.

### Motivation and context

Closes #477 

### Screenshots, videos, or demo, if appropriate


https://477-code-block-max-height--mineral-ui.netlify.com/

### How to test


1. `cd mineral-ui && npm start`
2. Go to Getting Started
3. Make sure you don't have to scroll the code block
4. Go to a component demo with a long source (MenuItem's Custom Render is good)
5. Make sure you _do_ have to scroll that code block

### Types of changes

- Other (provide details below)

Just a site thing.

### Checklist
